src/main/java/carpet/CarpetSettings.java

```diff
@@ -652,4 +650,4 @@ public static void changeSpawnSize(int size)
             validate = Validator.PROBABILITY.class
     )
     public static double creativeFlyDrag = 0.09;
-}
+}
```

Comment:
while at it, you might revert all the changes to CarpetSettings - not to introduce unnecessary diffs.

Reply:
done